src/core/public/doc_links/doc_links_service.ts

```diff
@@ -164,98 +421,214 @@ export interface DocLinksStart {
+      readonly commonParameters: string;
+      readonly popularAPI: string;
+      readonly restAPI: string;
     };
-    readonly functionbeat: {
-      readonly base: string;
-    };
-    readonly winlogbeat: {
-      readonly base: string;
-    };
-    readonly aggs: {
-      readonly date_histogram: string;
-      readonly date_range: string;
-      readonly filter: string;
-      readonly filters: string;
-      readonly geohash_grid: string;
-      readonly histogram: string;
-      readonly ip_range: string;
-      readonly range: string;
-      readonly significant_terms: string;
-      readonly terms: string;
-      readonly avg: string;
-      readonly avg_bucket: string;
-      readonly max_bucket: string;
-      readonly min_bucket: string;
-      readonly sum_bucket: string;
-      readonly cardinality: string;
-      readonly count: string;
-      readonly cumulative_sum: string;
-      readonly derivative: string;
-      readonly geo_bounds: string;
-      readonly geo_centroid: string;
-      readonly max: string;
-      readonly median: string;
-      readonly min: string;
-      readonly moving_avg: string;
-      readonly percentile_ranks: string;
-      readonly serial_diff: string;
-      readonly std_dev: string;
-      readonly sum: string;
-      readonly top_hits: string;
-    };
-    readonly scriptedFields: {
-      readonly scriptFields: string;
-      readonly scriptAggs: string;
-      readonly painless: string;
-      readonly painlessApi: string;
-      readonly painlessSyntax: string;
-      readonly luceneExpressions: string;
-    };
-    readonly indexPatterns: {
-      readonly loadingData: string;
+    readonly opensearchDashboards: {
       readonly introduction: string;
+      readonly installation: {
+        readonly base: string;
+        readonly docker: string;
+        readonly tar: string;
+        readonly helm: string;
+        readonly tls: string;
+        readonly plugins: string;
+      };
+      readonly mapTiles: string;
+      readonly ganttCharts: string;
+      readonly reporting: string;
+      readonly notebooks: {
+        readonly base: string;
+        readonly notebook_tutorial: string;
+        readonly paragraph_tutorial: string;
+        readonly sample_notebook: string;
+        readonly create_report: string;
+      };
+      readonly dql: {
+        readonly base: string;
+        readonly terms_query: string;
+        readonly boolean_query: string;
+        readonly date_query: string;
+        readonly nested_query: string;
+      };
     };
-    readonly addData: string;
-    readonly opensearchDashboards: string;
-    readonly siem: {
-      readonly guide: string;
-      readonly gettingStarted: string;
-    };
-    readonly query: {
-      readonly eql: string;
-      readonly luceneQuerySyntax: string;
-      readonly queryDsl: string;
-      readonly kueryQuerySyntax: string;
-    };
-    readonly date: {
-      readonly dateMath: string;
+    noDocumentation: {
```

Comment:
Is there a reason this isn't `readonly`?

Reply:
Should mark as readonly as well